db_instance/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "mcs_db_instance" "main" {
+  name = var.db_instance_name
+
+  keypair = var.db_instance_keypair
+
+  datastore {
+    type    = "mysql"
+    version = "5.7"
```

Comment:
5.7 is a quite old version of MySql, why do we use it?

Reply:
add it as separate variable, by default v 8.0